src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/GlobalRunSpec.cs

```diff
@@ -3,21 +3,38 @@
 using System.Globalization;
 using System.Linq;
 using System.Text;
-using System.Text.RegularExpressions;
 using Microsoft.TemplateEngine.Abstractions;
 using Microsoft.TemplateEngine.Abstractions.Mount;
 using Microsoft.TemplateEngine.Core;
 using Microsoft.TemplateEngine.Core.Contracts;
-using Microsoft.TemplateEngine.Core.Expressions.Cpp;
 using Microsoft.TemplateEngine.Core.Operations;
 using Microsoft.TemplateEngine.Core.Util;
+using Microsoft.TemplateEngine.Orchestrator.RunnableProjects.Config;
 using Microsoft.TemplateEngine.Utils;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.TemplateEngine.Orchestrator.RunnableProjects
 {
     public class GlobalRunSpec : IGlobalRunSpec
     {
+        private static readonly IReadOnlyList<IOperationConfig> OperationConfigReaders;
+
+        static GlobalRunSpec()
+        {
+            List<IOperationConfig> operationConfigReaders = new List<IOperationConfig>
+            {
+                new ConditionalConfig(),
+                new FlagsConfig(),
+                new IncludeConfig(),
+                new MacrosConfig(),
+                new RegionConfig(),
+                new ReplacementConfig()
+            };
+
+            operationConfigReaders.Sort((x, y) => x.Order.CompareTo(y.Order));
+            OperationConfigReaders = operationConfigReaders;
+        }
+
```

Comment:
Do we want to allow configuration of the list of operationConfigReaders, as opposed to having they explicitly stated in the static constructor?